src/refactor_nrepl/ns/libspecs.clj

```diff
@@ -88,10 +88,10 @@
    {:clj  (->> (core/find-in-project (util/with-suppressed-errors
                                        (some-fn core/clj-file? core/cljc-file?)
                                        ignore-errors?))
-               (map (juxt identity (partial get-libspec-from-file-with-caching :clj)))
+               (pmap (juxt identity (partial get-libspec-from-file-with-caching :clj)))
```

Comment:
I'd add a note about the use of `pmap`, as people tend to "optimize" it by removing it. :D